src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb

```diff
@@ -0,0 +1,1412 @@
xg9+8IPirlR0kiZ72y/pPul+7Ze0beedd26/qfg6oDfffLP42X7HkUceGccff3x5U+nfh3/+85/xhS98IdITUXqyKDzpiVJ2jpGbZKcvREKAAAECrSsgN8lX3+c9H+lrLpJ6q175yLXXXisXyde/Hn2KVm7SJz4nEyBAgACBqgi0Wm6S+8KTrh4bt9tu28YF406pyg/CRQgQIECAAIFZC4wes39ceOFlFQem17Wk9wpbsi3w6quvFp8i0v7pGwsvvHDxqSXpFTWlpWPBSCrUSE/uSMu7775bLAb++9//Xjq8+JleQXPhhRfGHHPMUd7e8TqlHen1Nptsskmsssoq0b9//7jiiivizjvvLO0ufqZCmPavuUkb06t9UiFH+1fapO1bbbVV8ekiqcAjFSlPmDAhbS4vqY2pSGbgwIHF+NPnhx9+WN6fXqlzzjnnlNfTl7Fjx8b5559fsS2t3HLLLcUnwrTfsdZaa8U99/z31VPp8dm/+MUvYv/9949TTz21fGi/fv3ioIMOiq985SuRvqf+ePbZZ4sx33fffZH+8cSTMlfmv8hNMt9FAiRAgACBFhCQm+Srk/Ocj1QjF0m9lfKpeuQjM2bMkIvk61+PPkUrN+kTn5MJECBAgEBVBFotN8l94YnHxlXld+8iBAgQIECgTwKt9Li4PkFl9OTRo0cXC0RK4R188MHxs5/9rLRa/OxYMFIqPEmFHZtvvnncfPPNFcdvvfXW8cc//rFYTNF+R8frpAnWww8/PPbYY49oX+iSHlm97rrrxl133dX+9Hj77bdjoYUWKm/bcMMNi4UfpQ1zzz13pAm2nXbaqbSp+PmTn/wkfvjDH1Zs23PPPYuv3Ukbv/nNb8af/vSn8v4VV1yxWJhS3tD2Jb02Kr0PveNyxBFHxHHHHVfenGJMr/RpXwxz9913x5prrllsU/uCmu985zvx29/+tnxu+y/ptUGpPekJNJZ8CMhN8tFPoiRAgACB5haQm+Svf/Oaj1QrF0k9Vo985MADD6wo7peL5O/fldmJWG4yO1qOJUCAAAECtRFotdwk94Un6WfQ1WPjjjpq/zj6qANq8ytxVQIECBAgQKAs0NXj4tLOVDhgyY/Agw8+WHxCRyoM+cxnPtMp8I4FI6nwJL16JhV4/OEPf6g4/mtf+1pcffXVMWDAgIrtaaWr65SenNLx4DPPPDPSq3raL5MmTYpll122uCk9GSQViLRfdt1114oimvb71l577UgFIKUlFXWk4o70pJFU/JHObb+0v9czzzwTK620Uvvd5e/puu2LSVIBS5o4Li0p3lSwkp78stFGG8X48eNLu4r3TE+FsTSPgNykefpSSwgQIEAgfwJyk/z1WSnivOUj1cxFkkE98pGUp8lFSr+41viUm7RGP2slAQIECGRToBVzk6YoPEmP8T/mmGMqflVDhizd9kj1P8byQ/7zBxMVO60QIECAAAECVRPoqmrXa3aqxpuZC3VVMJJej/OrX/2qIsZUhHHjjTfG/PPPX7G9tNLVdborPOlYwJGu0b4Y5Prrr4/NNtusdOni57333lt8xU7Fxv+/8rvf/S7S3+prv6SCkmHDhsVbb70VqT3tXzmUCkJKxSi//vWvY6+99iqemp5AsuOOO5afVJLW33zzzVhggQWK+zu+5/1///d/45RT/vMayPQkmUMPPbQcQipGSX/DMz2NpWMRTfkgX3IlIDfJVXcJlgABAgSaTEBu0mQd2q45XeURjcxHqpmLpGbWIx+Ri7T7QbXIV7lJi3S0ZhIgQIBAJgVaMTdpisKTrh4bl35hu+22bVww7j+T/Jn8xQmKAAECBAjkXKCrdxSmJj3//PPF15LkvHnCbyfQcaK33a7y1xEjRrQV/k6IhRdeuLyt45eO1ym9sqfjcWn9L3/5S/E1Pu33tS88SUUvP/jBD9rvjg8//LDLJ62kgx599NFYddVVK46/7rrrYtNNNy1uS5833HBDeX8qCBk3blxxffvtty++OiitpOKa9Jjqb33rW+VjU6ylIpiVV145nnjiifK+O+64I9ZZZ53ieophtdVW6/REoNLTUMaOHVt8WkpXT4spX9CXTAvITTLdPYIjQIAAgSYWkJs0cee2Na1jHtFVa+uZj1Q7F0ntqXU+Ihfp6lfT3NvkJs3dv1pHgAABAtkVaNXcpCkKT9LPqrvq3QsuOCXWH7lWdn95IiNAgAABAjkVeGHi5Bg6dO1O0XvaSSeSptgwq4neT33qU/HUU0/FwIEDZ9rejtfpS+FJKjpp/8SVRRddNF5//fVu7//GG2/EYostVrH/tNNOi/SEkrT85je/iT322KO8f7nlliu+Iie9Nir9bcrXXnutuO/II4+M/fffPxZffPGYMWNGcVsqRDnppJPiX//6V/lVQGnH0ksvHZMnTy6+Zqd4YNv/XHrppW0F0rsVi2RK29p/Dho0KA455JDYe++9uy2iaX+879kTkJtkr09ERIAAAQLNLSA3ae7+Ta3rmEd0bHG985Fq5yKpPfXIR+QiHX85zb8uN2n+PtZCAgQIEMiWQCvnJk1TeNJd9e7IkWvGhFv/mK1fnGgIECBAgEATCKw/aru47bZ7OrUk/SG9pfkEZjXRm57YccEFF5RfTdOdQMfr9KXwZM8994xzzjmnfKsll1wyXnrppfJ6xy/vvfde+XU4pX2pWCQVjaRlypQpsdRSS8X06dNLuyO9u/3tt9+OL3zhC+Vt6akuI0eOjC996Utx//33F7en/X//+98jvZ4nPSmltKSillTc0nFJrwRKxSdPPvlkx13l9TXXXDOuueaaTsUy5QN8yayA3CSzXSMwAgQIEGhSAblJk3Zsu2Z1zCPa7Sp+rXc+Uu1cJDWiXvmIXKTjr6e51+Umzd2/WkeAAAEC2RNo5dykaQpP0s8q/WHHmDFjOv3Cjjpq/zj6qAM6bbeBAAECBAgQ6J1Ad4+KS68laf+H7r27urOyKNBxojc97WPq1Kkxbdq0crhzzTVXXHHFFbHF
```

Comment:
Would be nice to have a graphic of this too

Reply:
I think this data is a bit hard to relate to for a beginner who doesn't know about "TheSemanticWeb" and the related books.